src/ManifoldsBase.jl

```diff
@@ -140,6 +140,23 @@ Compute the angle between tangent vectors `X` and `Y` at point `p` from the
 function angle(M::AbstractManifold, p, X, Y)
     return acos(real(inner(M, p, X, Y)) / norm(M, p, X) / norm(M, p, Y))
 end
+
+"""
+    are_linearly_independent(M::AbstractManifold, p, X, Y)
+
+Check is vectors `X`, `Y` tangent at `p` to `M` are linearly independent.
+"""
+function are_linearly_independent(M::AbstractManifold, p, X, Y)
+    eps_abs = sqrt(eps(number_eltype(X)))
```

Comment:
would it maybe make sense to make this a `tolerance` keyword argument?

besides that this function should stay internal (not exported) since it is only an internal helper.

Reply:
That's a good idea. I've kept the name to indicate that it's absolute tolerance (we might also want to specify relative tolerance for the `isapprox` test).